bitnami/rabbitmq/Chart.yaml

```
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/bitnami-docker-rabbitmq
   - https://www.rabbitmq.com
-version: 8.31.4
+version: 8.31.5
```

The changes look good @Tajjar. Before we merge please bump the chart's version to a new minor, as this is a feature addition.
```suggestion
version: 8.32.0
```

---

done